src/molecules/Content/ContentHeader.js

```diff
@@ -4,6 +4,49 @@ import Link from 'next/link';
 import Heading from '../../atoms/Heading/Heading';
 import TagLink from '../../atoms/TagLink/TagLink';
 
+const toByline = (authors) => {
+  if (!authors || authors.length <= 0) return;
+
+  if (authors.length === 1) {
+    return (
+      <div className="content_byline" data-testid="contentByline">
```

Comment:
could this go outside the if statement? Might be better

Reply:
It ends up being less DRY, in my opinion. It means I have to nest return statements, and I still have to use `<></>` tags in all the places where the `content_byline` divs were before. So yeah, not sure I want to do this change. Thanks for the other suggestions though!